website/content/docs/nia/usage/run-ha.mdx

```diff
@@ -0,0 +1,184 @@
+---
+layout: docs
+page_title: Run Consul-Terraform-Sync with High Availability
```

Comment:
Should the page title include an enterprise tag?

Reply:
Do you mean that enterprise alert tag, like on this page? https://www.consul.io/docs/nia/network-drivers/terraform-cloud

Reply:
I think what Lorna mentioned below (The enterprise message at the top of the page)